pkg/agent/core/ngt/service/ngt.go

```diff
@@ -235,18 +308,18 @@ func (n *ngt) Start(ctx context.Context) <-chan error {
 			err = nil
 			select {
 			case <-ctx.Done():
-				err = n.CreateIndex(ctx, n.dps)
+				err = n.CreateIndex(ctx, defaultPoolSize)
```

Comment:
I mean it's better to set this poolSize from config variable

Reply:
i think it's redundant to have same variables in both [internal/core/ngt.ngt](https://github.com/vdaas/vald/blob/ec75fbaed4e3e9808baee01f33d1e08e1443928d/internal/core/ngt/ngt.go#L88) and pkg/agent/core/ngt/service.NGT. so how about to add a new receiver function `CreateIndexWithDefaultPoolSize` to internal/core/ngt.NGT and call it from here?